components/atoms/Select.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const StyledSelectContainer = styled.div`
+`
+
+const StyledSelect = styled.select`
+  font-size: 16px;
+  padding: 10px 40px 10px 15px;
+  border-radius: 2em;
+  cursor: pointer;
+  background-color: ${props => props.theme.colors.gray1};
+  border: none;
+  outline: none;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAh0lEQVQ4T93TMQrCUAzG8V9x8QziiYSuXdzFC7h4AcELOPQAdXYovZCHEATlgQV5GFTe1ozJlz/kS1IpjKqw3wQBVyy++JI0y1GTe7DCBbMAckeNIQKk/BanALBB+16LtnDELoMcsM/BESDlz2heDR3WePwKSLo5eoxz3z6NNcFD+vu3ij14Aqz/DxGbKB7CAAAAAElFTkSuQmCC');
```

Comment:
I am not sure what this image is, but we should probably be either injecting the data-uri using some babel plugin or reference an image path and defer the decision of using a data-uri or something else to the webpack of whatever is using it.

Reply:
Another thing to keep in mind when using rasters is that it's usually best to have versions of the raster at 2x or 3x the resolution to support high density displays. For that reason it's actually probably best to have a vector image be the source and then using some tool to convert it to a raster.